### PR TITLE
Support ePix100 detector in EXtra-foam

### PR DESCRIPTION
**Pull request by zhujun98**

### CODE REVIEW

**Comment by zhujun98** [comment]
@ebadkamil can you check it when you still have the fresh memory from yesterday? Thank you!

File: extra_foam/offline/file_server.py [hunk 1]
```diff
@@ -225,8 +225,10 @@ def run(self):
             fast_devices = [("*DET/*CH0:xtdf", "image.data")]
         elif detector == "JungFrau":
             fast_devices = [("*/DET/*:daqOutput", "data.adc")]
-        elif detector == "FastCCD":
+        elif detector in ["FastCCD"]:
```
Comment (ebadkamil) [inline extra_foam/offline/file_server.py]: `== "FastCCD"`

File: extra_foam/offline/file_server.py [hunk 2]
```diff
@@ -225,8 +225,10 @@ def run(self):
...
             fast_devices = [("*/DAQ/*:daqOutput", "data.image.pixels")]
+        elif detector in ["ePix100"]:
```
Comment (ebadkamil) [inline extra_foam/offline/file_server.py]: `== "ePix100"
`

Just to be consistent

File: extra_foam/offline/file_server.py (hunk 1 above)
Comment (zhujun98) [reply extra_foam/offline/file_server.py]: It is all 'in' in the new commit. I hope the detector people will not invent more naming convention.

**Comment by zhujun98** [comment]
Thanks for the review!

**Pull request merged**